packages/block-library/src/latest-posts/edit.js

```diff
@@ -483,19 +483,27 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 								.split( ' ', excerptLength )
 								.join( ' ' ) }
 							{ createInterpolateElement(
-								/* translators: excerpt truncation character, default …  */
-								__( ' … <a>Read more</a>' ),
+								sprintf(
+									/* translators: %1$s is a static string "Read more", and %2$s is the post title only visible to screen readers. */
+									__( '… <a>%1$s<span>: %2$s</span></a>' ),
+									__( 'Read more' ),
+									titleTrimmed
```

Comment:
Does this need a fallback like we use for displaying the title? I.e.

```suggestion
									titleTrimmed || __( '(no title)' )
```